spec/spec_helper.rb

```diff
@@ -29,7 +29,6 @@
 
 RSpec.configure do |config|
   config.before(:suite) do
-    Webservices::Application.model_classes.each(&:recreate_index)
```

Comment:
I take back the one liner, but those were bugs, so I guess a fix is warranted regardless. :P